cube/src/test/java/zone/cogni/asquare/cube/convertor/ModelToJsonConversionTest.java

```diff
@@ -291,6 +292,32 @@ public void context_test() {
     assertThat(navigate(json, "data", "attributes", "person:name")).isNotNull();
   }
 
+  @Test
+  public void datetime_parse_ok() {
+    // given
+    Model homerModel = JenaUtils.read(new ClassPathResource("convertor/person-data-homer-datetime-parse.ttl"));
+    ModelToJsonConversion conversion = getContextPersonConversion(getContextConfiguration());
+
+    // when
+    ObjectNode json = conversion.apply(homerModel, "http://demo.com/data#homer");
+
+    System.out.println(json.toPrettyString());
+
+    // then
+    assertThat(navigate(json, "data", "attributes", "person:hasDate", "xsd:dateTime")).hasSize(3);
+  }
+
+  @Test
+  public void datetime_parse_not_ok() {
+    // given
+    Model homerModel = JenaUtils.read(new ClassPathResource("convertor/person-data-homer-datetime-parse-bad-format.ttl"));
+    ModelToJsonConversion conversion = getContextPersonConversion(getContextConfiguration());
+
+    // when
+    // then
+    assertThatExceptionOfType(RuntimeException.class).isThrownBy(()-> conversion.apply(homerModel, "http://demo.com/data#homer"));
```

Comment:
you can also check part of the contents of the exception. just to make the test a bit clearer... and to avoid any random exception is making the test pass. use `.withMessageContaining`